pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>com.conveyal</groupId>
   <artifactId>gtfs-lib</artifactId>
-  <version>4.2.5-SNAPSHOT</version>
+  <version>4.8</version>
```

Comment:
???

Reply:
This shouldn't have been checked in, sorry about that. In order to use a development copy of gtfs-lib with datatools-server, I manually change the version in the pom to a non-existent future version to match datatools-server so that Intellij will use the local code rather than a jar from the maven repo.